test/integration/BraveAlerterConfiguratorTest/getAlertSessionTest.js

```diff
@@ -50,6 +54,7 @@ describe('BraveAlerterConfigurator.js integration tests: getAlertSession', () =>
   afterEach(async () => {
     await db.clearSessions()
     await db.clearLocations()
+    await db.clearClients()
```

Comment:
I'm thinking about combining these int a `db.clearDatabase()` function that will do all this and then getting rid of these table-specific versions. It would make sure that we always do all of them and always in the right order (now that there are some foreign key constraints on them). What do you think? I won't do it as part of this PR, but for whenever one of us is in here making changes again.

Reply:
I think that's a good idea.